docs/response-parameters.rst

```diff
@@ -51,10 +51,12 @@ Descriptions of the custom response parameters that are marked with a leading ``
 * ``@changesetId`` - id of the OSM changeset where the contribution was performed
 * ``@osmType`` - type of the OSM element (NODE, WAY or RELATION)
 * ``@timestamp`` - indicates when this contribution occurred
-* ``@creation``	- contribution type; indicates if this feature is newly created (true); cannot occur in combination with other contribution types
+* ``@creation``	- contribution type; indicates if a new feature gets created OR moved into your requested area of interest OR is now fitting to your defined filter parameter (true); cannot occur in combination with other contribution types
 * ``@geometryChange`` - contribution type; indicates if the geometry of this feature has changed (true); can occur in combination with @tagChange
-* ``@tagChange``- contribution type; indicates if the tag of this feature has changed (true); can occur in combination with @geometryChange
-* ``@deletion`` - contribution type; indicates if the feature is deleted (true); cannot occur in combination with other contribution types
+* ``@tagChange``- contribution type; indicates if the tag(s) of this feature has/have changed (true); can occur in combination with @geometryChange
```

Comment:
IMHO there is no need to complicate the sentence for the special-case elements with a single tag.

```suggestion
* ``@tagChange``- contribution type; indicates if the tags of this feature have changed (true); can occur in combination with @geometryChange
```

Reply:
I'm fine with only using the plural form too. The single-form-only was not fitting too well imo, that's why I updated it.